src/worldcereal/rdm_api/rdm_interaction.py

```diff
@@ -0,0 +1,143 @@
+"""Interaction with the WorldCereal RDM API. Used to generate the reference data in geoparquet format for the point extractions."""
+
+from pathlib import Path
+from typing import List, Union
+
+import duckdb
+import geopandas as gpd
+import requests
+from shapely import Polygon, wkb
+
+# Define the default columns to be extracted from the RDM API
+DEFAULT_COLUMNS = [
```

Comment:
We propose these as the defaults: `['sample_id', 'ewoc_code', 'valid_time', 'quality_score_lc', 'quality_score_ct']`